Build Fails During Unit Tests

**Describe the bug**
When running a GA build of IP2CR, the build fails at the unit test stage with the following error:

```
Error: src/svc/ip_fuzzing/ip_fuzzing_test.go:7:2: package slices is not in GOROOT (/opt/hostedtoolcache/go/1.20.7/x64/src/slices)
Error: Process completed with exit code 1.
```

**To Reproduce**
Run oldstable build job

**Expected behavior**
Build completes.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Environment:**

Github Actions

**Additional context**
Add any other context about the problem here.

--- a/go.mod
+++ b/go.mod
@@ -11,6 +11,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/iam v1.22.5
 	github.com/aws/aws-sdk-go-v2/service/organizations v1.20.4
 	github.com/sirupsen/logrus v1.9.3
+	golang.org/x/exp v0.0.0-20230817173708-d852ddb80c63
 )
 
 require (

--- a/src/svc/ip_fuzzing/ip_fuzzing_test.go
+++ b/src/svc/ip_fuzzing/ip_fuzzing_test.go
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"testing"
 
-	"slices"
+	"golang.org/x/exp/slices" // Update to the stable `slices` package once 1.12 becomes oldstable ( Issue #112 )
 
 	ipfuzzing "github.com/magneticstain/ip-2-cloudresource/src/svc/ip_fuzzing"
 )

